test/integration/Proposal.test.ts

```diff
@@ -255,8 +254,8 @@ describe('Proposal', () => {
         const result = proposalModel.toJSON();
 
         expect(result.submitter).toBe(testData.submitter);
-        expect(result.blockStart).toBe(testData.blockStart);
-        expect(result.blockEnd).toBe(testData.blockEnd);
+        expect(result.startTime).toBe(testData.startTime);
+        expect(result.endTime).toBe(testData.endTime);
```

Comment:
this cant work, in your current implementation result doesn't contain startTime or endTime.

Reply:
Removed